### PR TITLE
Provide masks DDD and DDDD to print "Yesterday", "Today" or "Tomorrow".

### PR DESCRIPTION
**Pull request by TalhaAwan**
- If the date belongs to yesterday, today or tomorrow the masks should print so.  
- If the date does not lie in these days then DDD should fall back to ddd and DDDD to dddd.
- Unit tests added
- Readme updated

### CODE REVIEW

**Comment by TalhaAwan** [comment]
Though tests are passing, the logic is incorrect. I mistakenly used day of week instead of day of month. Even with day of month, boundary conditions on changing months and years will need to be handled. Will try to work on this later.

**Comment by TalhaAwan** [comment]
Fixed with simplified logic.

**Comment by chase-manning** [comment]
@TalhaAwan Thanks so much for this PR, is a great idea!!!  
Sorry this was not merged sooner, this repo was not under active maintenance at the time.  
I would love to merge this PR but it has merge conflicts at the moment.  
Would you be able to please fix the merge conflicts so I can merge this?

File: Readme.md [hunk 1]
```diff
@@ -82,7 +82,9 @@ dateFormat(now, "N");
 | `d`              | Day of the month as digits; no leading zero for single-digit days.                                                                                            |
 | `dd`             | Day of the month as digits; leading zero for single-digit days.                                                                                               |
 | `ddd`            | Day of the week as a three-letter abbreviation.                                                                                                               |
+| `DDD`            | "Yesterday", "Today" or "Tomorrow" if date lies within these three days. Else fall back to ddd.                                                               |
```
Comment (chase-manning) [inline Readme.md:85]: Please can we change the `DDD` Mask to show `Yes`, `Tod` & `Tom`. This is because the `ddd` mask which it defaults to always shows 3 characters. And so the user would likely be designing their UI around expecting three characters.

File: lib/dateformat.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
+/*
```
Comment (chase-manning) [inline lib/dateformat.js:1]: This change is introducing a new file `dateformat.js` under the `lib` directory.  However, the project is built around using the `dateformat.js` file in the `src` directory (see the main source code for reference).  Instead of creating a new file, can we please change this PR to make changes to the exiting `src/dateformat.js` file 😄

**Review by chase-manning — CHANGES_REQUESTED** [review]
Thanks so much for the PR!!!  
I have added some review comments to be resolved.

**Comment by TalhaAwan** [comment]
@chase-manning I resolved the readme conflicts only to find out that tests are not passing. Lots have changed since I opened this PR, so I'll have to re-add this functionality. Will do that soon.

**Review by chase-manning — APPROVED** [review]
Looks great!!! ❤️

**Comment by TalhaAwan** [comment]
@chase-manning "Yes", "Tom" and "Tod" look odd though. Do you think we should change that? I was looking it up and YDA, TDA, and TMW/TMR are the abbreviations that are used in certain fields. Let me know if this needs change.
